README.md

```diff
@@ -113,6 +111,8 @@ For example, the following body will try to send a german email:
 
 If the `lang` property is not present in the body of your request, the default email will be sent.
 
+If request contains `"lang": "de"` and one `de.txt`, `de.subject` or `de.mjml` file missing then `default.*` file will be used.
+
```

Comment:
@stephane-klein suggestion:

```
Inside your mail templates folders, you can create multiple languages files for the same email, in the following format: `{lang}.(mjml|subject|txt)`.

So, for example, an emails that supports french, english, and defaults to english, will have the following files in its template folder:

-- ```
$ ls mail-templates/mail-example
default.mjml -> en.mjml             fr.mjml          en.mjml
default.subject -> en.subject       fr.subject       en.subject
default.txt -> en.mjml              fr.txt           en.txt
-- ```

When you send the `lang` property to the body of your request, it will pick the language, and if not found, will fallback to default.{txt,mjml,subject}

For example, the following body will try to send a german email, or fallback to the default email if there are no german files in the template folder:

-- ```
{
  "from": "no-reply@example.com",
  "to": "john.doe@example.com",
  "lang": "de"
}
-- ```

#### Examples

Adding a french-only email
- create fr.{txt,mjml,subject} files in the email template folder
- run the following command to fallback to french: `ln -s FR.subject default.subject ; ln -s FR.txt default.txt ; ln -s FR.mjml default.mjml`

Create an english version, but keep french as the default
- create en.{txt,mjml,subject} files in the email template folder

Swith the default version to the english version
- `rm -f default.*`
- `ln -s EN.subject default.subject ; ln -s EN.txt default.txt ; ln -s EN.mjml default.mjml`

If any of the three language files is missing, gibbon-mail will fallback to the default version.